src/core/Bari.Core/cs/Model/Parameters/InheritableProjectParameters.cs

```diff
@@ -67,7 +67,7 @@ public object GetDynamic(string name, Type t)
 
             var value = GetPropertyValue(name);
             if (!value.IsSpecified && parent != null)
-                value = parent.GetPropertyValue(name);
+                return parent.GetDynamic(name, t);
```

Comment:
If the parameter is in suite level the original version couldn't find it. There is module, product and it's parent parameters is the suite level parameters.